docs/application/tizen-studio-0/setup/test.md

```diff
@@ -0,0 +1,135 @@
+
+The following figures shows how to set the `Colomns` and the `Rows` properties. Since the `GridOrientation` is set to horizontal, the row index of last item can be `2` that exceeds the `Rows` property.
+
+![Column](./media/columnLayout.png)
+
+```csharp
+View layoutView = new View();
+var gridLayout = new GridLayout();
+gridLayout.GridOrientation = GridLayout.Orientation.Horizontal;
+gridLayout.Columns = 2
+gridLayout.Rows = 1
+layoutView.Layout = gridLayout;
+```
+<a name="grid-orientation"></a>
+## Grid Orientation
+The `GridOrientation` only works when a child is added without the specified `Row` or `Column` property.
+ - On horizontal orientation, the children will autmatically occupy empty cells from left to right.
+ - On vertical orientation, the children will automatically occupy empty cells from top to bottom.
+
+The following figure shows how the `GridOrientation` property works:
+
+![gridOrientation](./media/gridOrientatio.png)
+
+## Column spacing and row spacing
+
+The `ColumnSpacing` and `rowSpacing` properties are the distance between columns or rows. In order to set same sapce between the children, you can use `ColumnSpacing` and `rowSpacing` properties instead of setting the same margin for each child.
+
+The following figure shows how the `ColumnSpacing` and `RowSpacing` properties work:
+
+![gridSpacing](./media/gridSpacing.png)
+
+The following table describes the properties that affect the specified child.
+
+| Property               | Type            | Description |
+| -----------------------| --------------- | ----------- |
+| `Column`               | int             | Gets or sets the column index of the child. |
+| `Row`                  | int             | Gets or sets the row index of the child. |
+| `ColumnSpan`           | int             | Gets or sets the number of columns that the child occupy. |
+| `RowSpan`              | int             | Gets or sets the number of rows that the child occupy. |
+| `HorizontalStretch`    | StretchFlags    | Gets or sets the value how child will be resized within horizontal space of the cell where child occupy. |
+| `VerticalStretch`      | StretchFlags    | Gets or sets the value how child is resized within vertical space of the cell where child occupy. |
+| `HorizontalAlignment`  | Alignment       | Gets or sets the horizontal alignment of the child within the cell. |
+| `VerticalAlignment`    | Alignment       | Gets or sets the vertical alignment of the child within the cell. |
+
+## Column and row
+
+The `Column` and `row` properties specify the cell index of the child. If these properties aren't assigned, they are automaticaly set dependig on [`GridOrientation`](#grid-orientation), [`Columns`](#columns-and-rows) and [`Rows`](#columns-and-rows).
```

Comment:
Please change the first letter to upper case in the word "row".

Also, please write the word completely : "aren't" as "are not"

Spell error : change "dependig" as "depending"

Reply:
Check here:https://samsung.github.io/TizenFX/API9/api/Tizen.NUI.GridLayout.html#properties

Please use these parameter names from the API

Reply:
As both Columns and Rows are going to the same location, #columns-and-rows,
how about changing it to below?
[`Columns` and `Rows`](#columns-and-rows)